src/homepage/Components/Impact.js

```diff
@@ -14,45 +14,47 @@ export default class Impact extends Component {
           imgUrl: "image1.svg",
           para1: "Know right Opportunities",
           para2: "at right time!",
+          link: "/oppcalender"
         },
         {
           id: 2,
           name: "CodeMaps",
           imgUrl: "image2.svg",
           para1: "Learn everything the",
           para2: "right way!",
+          link: "/codemaps"
         },
         {
           id: 3,
           name: "Open-Source Journey",
           imgUrl: "image3.svg",
           para1:
             "Get your journey in open-source started with our open-source projects!",
-          para2: "",
+          para2: ""
```

Comment:
@harsha14gupta Though it's an optional change but ideally we should keep such hardcoded JSON out of the component. Can we have `ImpactData.js` file outside and import it here?